### DISCUSSION TITLE
Command failed: install-tool golang 1.20.0

### DISCUSSION THREAD

**Discussion by omercnet** [mend/renovate-ce-ee#277] [Q&A]
Recently I (sometimes) get this artifact error

I'm having a hard time understanding or reproducing this - any idea what might cause it?
The biggest conundrum is that my go.mod specifies go.19 so I'm not sure why it's even trying to install 1.20

### ⚠ Artifact update problem

Renovate failed to update an artifact related to this branch. You probably do not want to merge this PR as-is.

♻ Renovate will retry this branch, including artifacts, only when one of the following happens:

 - any of the package files in this branch needs updating, or 
 - the branch becomes conflicted, or
 - you click the rebase/retry checkbox if found above, or
 - you rename this PR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:

##### File name: go.sum

```
Command failed: install-tool golang 1.20.0
No /opt/buildpack/tools/golang/1.20.0/bin defined - aborting

```

**Comment by rarkins** [comment]
Which version of Renovate On-Prem are you running?

**Comment by omercnet** [comment]
3.0.0

**Comment by rarkins** [comment]
Please keep debug logs and then find the lines leading up to this error the next time it happens
